examples/rollbar-spring-boot-webmvc/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+rollbar.access_token = <ACCESS_TOKEN>
+rollbar.environment = development
+rollbar.framework = spring-webmvc
```

Comment:
I think this param is not used, and it will be filled up by our default config in Spring.